src/modules/mod.rs

```diff
@@ -28,6 +29,7 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         "git_status" => git_status::module(context),
         "username" => username::module(context),
         "battery" => battery::module(context),
+        "timer" => timer::module(context),
```

Comment:
Thoughts on naming this module `command_duration`?
I don't think timer is very clear.

Reply:
Agreed on `timer`. Would `cmd_duration` also work?

Reply:
Yep, that should be fine.